pkg/controller/servicemesh/webhooks/hacks.go

```diff
@@ -122,10 +119,11 @@ func createWebhookResources(ctx context.Context, mgr manager.Manager, log logr.L
 		return err
 	}
 
-	if err := registerConversionWebhook(ctx, mgr, log, operatorNamespace, &smcpConverterServicePath, webhookca.ServiceMeshControlPlaneCRDName); err != nil {
+	if err := RegisterConversionWebhook(ctx, cl, log, operatorNamespace, &smcpConverterServicePath, webhookca.ServiceMeshControlPlaneCRDName, true); err != nil {
 		return err
 	}
-	if err := registerConversionWebhook(ctx, mgr, log, operatorNamespace, &smeConverterServicePath, webhookca.ServiceMeshExtensionCRDName); err != nil {
+
+	if err := RegisterConversionWebhook(ctx, cl, log, operatorNamespace, &SmeConverterServicePath, webhookca.ServiceMeshExtensionCRDName, false); err != nil {
```

Comment:
do we need to check for `IsNotFound` error here, to prevent returning an error if the sme crd is not installed?